QuizApp/Presenters/Quiz/QuizPresenter.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  QuizPresenter.swift
+//  QuizApp
+//
+//  Created by Luka Namačinski on 12.05.2021..
+//
+
+import Foundation
+
+class QuizPresenter: QuizPresenterProtocol {
+    weak var coordinator: QuizzesCoordinator?
+    let dataService: NetworkServiceProtocol = NetworkService()
+    var delegate: QuizDelegate
```

Comment:
Isto kao i kod drugih Presentera. Molim te prođi kroz sve presentere i ispravi to.